tests/plan/show/test.sh

```diff
@@ -1,11 +1,46 @@
 #!/bin/bash
 . /usr/share/beakerlib/beakerlib.sh || exit 1
 
+function dump_fmf_id_block
+{
+    typeset output=${1?"*** output file"}
+    typeset lineno=$(cat -n $output | egrep 'fmf-id' | awk '{print $1}')
+    sed -n "$lineno,$"p $output
+}
+
 rlJournalStart
     rlPhaseStartSetup
         rlRun "pushd data"
         rlRun "set -o pipefail"
         rlRun "output=\$(mktemp)"
+        rlRun "show_tmp=\$(mktemp)"
+        rlRun "show_dir=\$(mktemp -d)"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Show a plan with -vvv in git repo"
+        rlRun -s "tmt plans show -vvv mini"
+        dump_fmf_id_block $rlRun_LOG > $show_tmp
+        rlRun "cat $show_tmp"
+        rlAssertGrep "url:" $show_tmp
+        rlAssertGrep "ref:" $show_tmp
+        rlAssertGrep "path:" $show_tmp
+        rlAssertGrep "name:" $show_tmp
+        rlAssertGrep "web" $show_tmp
+    rlPhaseEnd
+
+    rlPhaseStartTest "Show a plan with -vvv in non-git repo"
+        rlRun "pushd $(pwd)"
+        rlRun "cd $show_dir"
```

Comment:
Why not directly `rlRun "pushd $show_dir"`?  Pushd remembers which directory is currently in so the above line is not necessary.

Reply:
Updated. Thanks for the suggestion!